lib/beaker/subcommand.rb

```diff
@@ -61,5 +63,51 @@ def provision()
         say "Hosts have already been provisioned"
       end
     end
+
+    desc 'exec', 'execute a directory, file, or beaker suite'
+    long_desc <<-LONG_DESC
+    Run a single file, directory, or beaker suite. If supplied a file or directory,
+    that resource will be run in the context of the `tests` suite; If supplied a beaker
+    suite, then just that suite will run. If no resource is supplied, then this command
+    executes the suites as they are defined in the configuration.
+    LONG_DESC
+    def exec(resource=nil)
+      if options[:help]
+        invoke :help, [], ["exec"]
```

Comment:
Given the new way we're passing options to the subcommands, is my previous comment still valid? I.e. is the --help option still necessary here, or will we be able to pass that flag as part of the global options that we pass the subcommands? If so, should we remove the hidden --help flags from other subcommands, and then parse CLI options before checking for the --help flag?